### ISSUE TITLE
Roadmap: Homepage Manager

### ISSUE THREAD

**Issue by jobvector** [salesagility/SuiteCRM#579]
Hello,

are there features like:
- Moving/Reorganizing Tabs.
- Reassign Number of Columns on a tab for example from 3 Columns to 2 Columns.
- Rebuild Tabs (sometimes the dashboards gets corrupted and the whole tab must be rebuild)

for the homepage manager planed?

Do you have an estimation when the Homepage Manager will be implemented to suitecrm?

Thanks

**Comment by jobvector** [comment]
the suitecrm roadmap had radical changes and a lot of features are not listed, is there still a homepage (dashboard) manager planned?

There is also an old sugarcrm plugin that has some issues with suitecrm but had great functions:
https://github.com/geraldclark/DashboardManager
https://docs.google.com/document/pub?id=1uBxjbPcobsWahF46kLp51Ctz8himbufeij5O9OHvBrU

**Comment by Mausino** [comment]
:+1:

**Comment by ghermans** [comment]
:+1:

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Roadmap: Homepage Manager](https://trello.com/c/QngJl86z/142-roadmap-homepage-manager)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
